src/components/views/Articles/index.js

```diff
@@ -9,7 +9,7 @@ class Articles extends Component {
         const symbolsAmount = 300;
 
         return (
-            <section>
+            <div>
                 {posts.map(({ id, date, title, author, tags, content }) => {
```

Comment:
You can make this string more readable, moving parts of this code to the new lines